### PR TITLE
 Landing page (mobile formatting still needs work)

### PR DESCRIPTION
**Pull request by yaaowusu00**
What do you guys think of this design? I like how it looks on desktop (click the link to see video demo) , but when I play with mobile, things get a little bit off center and the text in the header overflows. I started trying to fix those mobile scaling issues but its taking longer than expected, so I think I'll have to finish the about page tomorrow and work on making the site look better on mobile next week.

https://drive.google.com/file/d/1ZgtnJFUbvBhl4aC9zlSo1FvarW1Uw3ej/view?usp=sharing

### CODE REVIEW

**Comment by yaaowusu00** [comment]
Here is what it looks like on iPad and iPhone as of now.
Screen Shot 2020-04-04 at 4 11 45 PM
Screen Shot 2020-04-04 at 4 12 01 PM

**Comment by SoyOscarRH** [comment]
Just my opinion:
* The general vibe the page is giving me is really great :D
* I really liked the use of multiple fonts for different things:
* I also am a big fan of the header

Maybe I could help you out tomorrow at some time to work on the mobile design
(I was hoping to not over complicate things for seem like the right tool for the job: search media queries, css)

(also, something really common on the industry is the idea of mobile first, because many people simply did not take enough time to think about how things are going to look on a mobile experience, so the idea is to always focus on mobile and because you are developing on a desktop you will inevitably have to develop a good view for you, https://www.google.com/search?q=mobile+first)

**Comment by SoyOscarRH** [comment]
Also I use this page for converting videos to gifs:
https://ezgif.com/video-to-gif

**Comment by SoyOscarRH** [comment]
More thoughts:
- The translucent header seems nice but could be really distracting sometimes, we could use some blur, so it is not opaque but not super distracting
Screen Shot 2020-04-04 at 10 47 07 PM

P.S. Seem super cool that you used a different header for mobile

* Maybe is just me (because I am daltonic, so I have truble with the red and brown and shades of purple) but the colors seems a bit off
 - Google have a great page for this: https://material.io/resources/color/
 - https://colorhunt.co/

Maybe we should discuss a color palette to use everywhere so the app stays consistent (we could use some time of the meeting for that)?

**Comment by SoyOscarRH** [comment]
But in general, great work, seems really to ship

**Comment by gilemos** [comment]
I also liked the design. I agree with Oscar that the opaque header is a little distracting and that a blur could be better. Moreover, the circles don't seem to be very centered in the video (but that is just a minor thing).

File: fudge-ratio/src/main/webapp/about_style.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  /* 161, 111, 32, 0.712*/
+  background-color: rgba(45, 93, 122, 0.712);
+  font-family: 'Poiret One', cursive;
+  position: fixed;
+  top: 0;
+  width: 100vw;
+  height: 15vh;
+}
+
+#logo{
```
Comment (gilemos) [inline fudge-ratio/src/main/webapp/about_style.css:26]: Just a small suggestion: what do you think about putting all the classes on top and the ids in the bottom (or the opposite) to make it easier to find?

**Review by gilemos — APPROVED** [review]
Good job Yaa! I left some comments, but it is approved :-)

File: fudge-ratio/src/main/webapp/about_style.css (hunk 1 above)
Comment (yaaowusu00) [reply fudge-ratio/src/main/webapp/about_style.css:26]: I can do that !

**Comment by yaaowusu00** [comment]
> Good job Yaa! I left some comments, but it is approved :-)

**Comment by yaaowusu00** [comment]
> I also liked the design. I agree with Oscar that the opaque header is a little distracting and that a blur could be better. Moreover, the circles don't seem to be very centered in the video (but that is just a minor thing).

Yeah, something's going on with the alignment of the circles and I haven't been able to figure it out yet.

**Comment by yaaowusu00** [comment]
> * Maybe is just me (because I am daltonic, so I have truble with the red and brown and shades of purple) but the colors seems a bit off
> 
> * Google have a great page for this: https://material.io/resources/color/
> * https://colorhunt.co/
> 
> Maybe we should discuss a color palette to use everywhere so the app stays consistent (we could use some time of the meeting for that)?

@SoyOscarRH  Personally, I like the colors on this page right now, but I agree that we should have a meeting on color palette.

**Pull request merged**